lesson05/script.js

```diff
@@ -5,17 +5,17 @@ const lists = [
    ]
 
 const promise = new Promise(resolve => {
-    resolve(lists)
+    resolve(lists);
 });
 
-promise.then(() => {
+promise.then((lists) => {
     const fragment = document.createDocumentFragment();
     for(const list of lists){
```

Comment:
Added variable to receive value.
Because we need to prepare a variable to store the value received by resolve.